src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
```

Comment:
It's recommended to use dependency injection or instantiate the service class outside of the controller... using dependency injection allows for easier testing and better separation of concerns. 

constructor(private readonly userService: UserService) {}

or 
private readonly userService: UserService;

constructor(userService: UserService) {
    this.userService = userService 
}